pallets/nft/src/lib.rs

```diff
@@ -221,6 +306,8 @@ pub mod pallet {
 		NFTTokenClassDestroyed(T::AccountId, T::ClassId),
 		NFTBoughtFromPool(T::AccountId, T::AccountId, T::ClassId, T::TokenId),
 		NFTSoldToPool(T::AccountId, T::AccountId, T::ClassId, T::TokenId),
+		NFTTokenPoolCreated(T::AccountId, T::ClassId),
+		NFTTokenPoolDestroyed(T::AccountId, T::ClassId),
```

Comment:
I would drop the NFT prefix from all events. When you get an event, you already see from which pallet it comes from.